e2e/packages/contracts/src/codegen/tables/Vector.sol

```diff
@@ -277,13 +304,13 @@ library Vector {
   }
 
   /** Tightly pack full data using this table's field layout */
-  function encode(int32 x, int32 y) internal pure returns (bytes memory) {
+  function encode(int32 x, int32 y) internal pure returns (bytes memory, PackedCounter, bytes memory) {
```

Comment:
does it help at all gas wise to name these return values and use them in the function body?

```solidity
function encode(int32 x, int32 y) internal pure returns (bytes memory _staticData, PackedCounter, bytes memory) {
  _staticData = encodeStatic(x, y);

```

Reply:
tested on `Callbacks` and `Vector2` but it doesn't seem to make a difference. Intuitively I'd also assume that this shouldn't affect bytecode